packages/toolkit/src/components/NewsLetterCard.tsx

```diff
@@ -0,0 +1,122 @@
+"use client";
+import { Icons } from "@instill-ai/design-system";
+import { ImageWithFallback } from "./ImageWithFallback";
+import { useState } from "react";
+import { useBlogPosts } from "../lib/react-query-service/misc/useBlogPosts";
+
+interface BlogPostData {
+  id: string;
+  imageUrl: string;
+  title: string;
+  publishedOn: string;
+  themeImgAlt: string;
+  themeImgSrc: string;
+  slug: string;
+}
+
+
+
+const NewsLetterCard = () => {
+  const { data: blogPosts = [], isLoading, isError } = useBlogPosts();
+
+  const [currentIndex, setCurrentIndex] = useState(0);
+
+  const NewsLetterCardSkeleton = () => {
+    return (
+      <div className="flex w-full flex-col px-2">
+        <div className="h-[250px] w-full animate-pulse bg-semantic-bg-secondary" />
+        <div className="mb-2 h-5 w-1/2 animate-pulse rounded bg-semantic-bg-secondary" />
+      </div>
+    );
+  };
+
+  const handlePrevious = () => {
+    setCurrentIndex((prevIndex) => (prevIndex === 0 ? 0 : prevIndex - 1));
+  };
+
+  const handleNext = () => {
+    setCurrentIndex((prevIndex) =>
+      prevIndex === blogPosts.length - 1 ? prevIndex : prevIndex + 1
+    );
+  };
+
+  if (isLoading) {
+    return <NewsLetterCardSkeleton />;
+  }
+
+  if (isError) {
+    return <div>Error fetching blog posts.</div>;
+  }
+
+  const { imageUrl, title, publishedOn, themeImgAlt, slug } =
+    blogPosts[currentIndex];
+
+  return (
+    <div className="flex h-[450px] flex-col gap-y-2 rounded-sm border border-semantic-bg-line p-4">
+      <h2 className="mb-4 text-2xl font-bold">What's New?</h2>
```

Comment:
@thewbuk Hi I forget to mention that we have design-token for typography too. 

Take this section for example, in design handover, the value is set to `Prod/Headings/Heading 3` the token will be `product-headings-heading-3`.